cloudify_cli/commands/logs.py

```diff
@@ -72,6 +74,7 @@ def _download_archive(conn, host_type, output_path, logger):
     conn.get(archive_path_on_host, output_path)
     logger.info('Removing archive from host...')
     conn.sudo('rm {0}'.format(archive_path_on_host))
+    output_json['archive paths'][conn.host] = output_path
```

Comment:
I don't think this would work in the 3-nodes case. You would end up having 3 archive paths instead of 9.